kratos.gid/apps/MPM/xml/ConstitutiveLaws.xml

```diff
@@ -212,6 +212,42 @@
     </outputs>
   </CLaw>
 
+  <!-- Borja Modified Cam Clay -->
+  <CLaw n="HenckyBorjaCamClayPlastic3DLaw" pn="Modified Cam Clay" class="HenckyBorjaCamClayPlastic3DLaw" Type="3D" LargeDeformation="True" kratos_module="ParticleMechanicsApplication"
+	help=" " Dimension="3D" ProductionReady="ProductionReady"  ImplementedInApplication="ParticleMechanicsApplication">
+    <inputs>
+      <parameter n="DENSITY" pn="Density" unit_magnitude="Density" units="kg/m^3" v="7850"/>
+      <parameter n="IS_IN_SITU" pn="In-situ case" type="bool" v="False"/>
+      <parameter n="INITIAL_MEAN_VOLUMETRIC_STRESS" pn="Initial Mean Volumetric Stress" unit_magnitude="P" units="Pa" v="-90.e3" />
+      <parameter n="OVER_CONSOLIDATION_RATIO" pn="Over Consolidation Ratio (OCR)" v="1.0"/>
```

Comment:
@jginternational 
I want to add the `INITIAL_MEAN_VOLUMETRIC_STRESS` and `OVER_CONSOLIDATION_RATIO` parameter option if the `IS_IN_SITU == false`. Otherwise, if `IS_IN_SITU == True`, the user doesn't have to specify the above parameters, and thus, it is better to hide them.

If such option is possible, can you tell me how to do that? Thanks :+1: